inginious/frontend/templates/course_admin/tags.html

```diff
@@ -0,0 +1,94 @@
+$def with (course, saved, error)
+
+$#
+$# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for
+$# more information about the licensing of this file.
+$#
+
+$var title: $:_("Tags") | $:course.get_name(user_manager.session_language())
+
+$var Column: $:template_helper.call('course_admin_menu',course=course,current='tags')
+$ is_admin = user_manager.has_admin_rights_on_course(course)
+
+$def NavbarF():
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <li class="breadcrumb-item"><a href="$get_homepath()/course/$course.get_id()">$course.get_name(user_manager.session_language())</a></li>
+            <li class="breadcrumb-item"><a href="$get_homepath()/admin/$course.get_id()" title=$:_('"Administration"') data-toggle="tooltip" data-placement="bottom">
+                <i class="fa fa-user-secret"></i></a></li>
+            <li class="breadcrumb-item active"><a href="#"><i class="fa fa-tags"></i> $:_("Tags") <span class="sr-only">$:_("(current)")</span></a></li>
+        </ol>
+    </nav>
+$var Navbar: $:NavbarF()
+
+<h2>$:_("Tags")</h2>
+$if error:
+    <div class="alert alert-danger" role="alert">
+        $error
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+$elif saved:
+    <div class="alert alert-success" role="alert">
+        $:_("Settings saved.")
+        <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+
+<form id="tags_form" method="post" enctype="multipart/form-data" class="form-horizontal">
+<table id="table" class="table">
+    <thead class="table-borderless">
+        <tr>
+            <th>$:_("id")</th>
+            <th>$:_("name")</th>
+            <th>$:_("description")</th>
+            <th>$:_("show to students")</th>
+            <th>$:_("type ")</th>
+        </tr>
+    </thead>
+    <tbody>
+
+    <!-- An empty row so that we can copy it to add new rows. All 'disabled' are important to not send this row to INGInious when saving the task-->
+    <tr id="NEW" style="display:none;">
+        <td><input disabled type="text" ID_STOP id="id_NEW" class="form-control" name="tags[NEW][id]" value="ID_REPLACE"></td>
+        <td><input disabled type="text" class="form-control" name="tags[NEW][name]" value="NAME_REPLACE"/></td>
+        <td><textarea disabled class="form-control" onfocus="studio_expand_tag_description(this);" onblur="studio_expand_tag_description_not(this);" name="tags[NEW][description]" rows="1" style="resize:none">DESCRIPTION_REPLACE</textarea></td>
+        <td><input disabled type="checkbox" class="form-control" name="tags[NEW][visible]" VISIBLE_REPLACE)/></td>
```

Comment:
Design remark : I think it's better to remove the form-control class here. I have a huge square:
![image](https://user-images.githubusercontent.com/13468630/59491420-ff35f980-8e86-11e9-973a-80ceec364633.png)